core/design/src/main/java/edu/stanford/spezikt/core/design/component/SpeziValidatedOutlinedTextField.kt

```diff
@@ -0,0 +1,96 @@
+package edu.stanford.spezikt.core.design.component
+
+import android.content.res.Configuration
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.OutlinedTextField
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.input.ImeAction
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.tooling.preview.PreviewParameterProvider
+import edu.stanford.spezikt.core.design.theme.SpeziTheme
+
+@Composable
+fun SpeziValidatedOutlinedTextField(
```

Comment:
ironically, i came up with the name because of MaterialTheme , which could just be called Theme, i guess, because it also comes from the package material3, but you're right it doesn't seem like they use that for UI components. so i'm happy to adapt it to your suggestion so it follows the guidelines